pycm/pycm_curve.py

```diff
@@ -96,9 +97,60 @@ def area(self, method="trapezoidal"):
                 raise pycmCurveError(AREA_METHOD_ERROR)
         return self.auc
 
-    def plot(self):
-        """Plot method."""
-        pass
+    def plot(
+            self,
+            classes=None,
+            area=False,
+            method="trapezoidal",
+            colors=None,
+            marker=None,
+            linewidth=1):
+        """Plot the given curve.
+
+        :param classes: ordered labels of classes
+        :type classes: list
+        :param area: area flag
+        :type area: bool
+        :param method: numerical integral technique (trapezoidal or midpoint)
+        :type method: str
+        :param colors: color for each class in plot
+        :type colors: list
+        :param marker: plot marker
+        :type marker: str
+        :param linewidth: plot line width
+        :type linewidth: float
+        :return: plot axes
+        """
+        try:
+            from matplotlib import pyplot as plt
+        except Exception:
+            raise pycmPlotError(MATPLOTLIB_PLOT_LIBRARY_ERROR)
+        if classes is None:
+            classes = self.classes
+        if area:
+            self.area(method=method)
+        if colors is not None and len(classes) != len(colors):
+            raise pycmPlotError(PLOT_COLORS_CLASS_MISMATCH_ERROR)
+        fig, ax = plt.subplots()
+        x, y = self.plot_x_axis, self.plot_y_axis
+        ax.set_xlabel(x)
+        ax.set_ylabel(y)
+        fig.suptitle("{} per {}".format(x, y))
+        for c_index, c in enumerate(classes):
+            label = "class {}".format(c)
+            if area:
+                label += "(area={:.3f})".format(self.auc[c])
+            color = None
```

Comment:
@sadrasabouri 

1. It seems here you overwrite the `turtle` color.
Am I right? Why?

2. What happens if the user does not enter any color?

Reply:
1. Ah! yeah. It was because of the VSCode suggestions.
2. no problem, `color=None` works the same as not setting any `color` parameter for the `plt.plot` method.